packages/contracts/contracts/ionic/CollateralSwap.sol

```diff
@@ -151,7 +150,7 @@ contract CollateralSwap is Ownable2Step, Exponential, IFlashLoanReceiver {
       if (!transferStatus) {
         revert TransferFailed(address(oldCollateralMarket), borrower, address(this));
       }
-      uint256 redeemResult = oldCollateralMarket.redeemUnderlying(borrowedAmount);
+      uint256 redeemResult = oldCollateralMarket.redeem(amountCTokensToSwap);
```

Comment:
@rhlsthrm isn't it better to pass `uint256 amountUnderlying` directly to the `swapCollateral` fn?